readme.md

```diff
@@ -164,12 +198,17 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 
 > Your answer...
 
+Nothing is defined above the yield tag that would be shared across all the views. It is completely empty so all
+the app is see is empty space between the empty opening and closing tags.  Also the yield tag is missing =
```

Comment:
Mostly correct. You're right that the yield is missing an `=`, but specifically there is something defined above the yield, but two of the tags are missing a `>` in their tag